src/data_processing/wiki_to_netflix.py

```diff
@@ -0,0 +1,136 @@
+import requests
+import csv
+
+base_dir = os.path.join(os.path.dirname(__file__), '../data/')
```

Comment:
This is `src/data` right? It probably makes more sense for the data to be at the top level (`MediaBridge/data`) since it's not source code, which `src` implies.

Reply:
Yeah, I'm using the `src/data` folder which was already there (but empty). I could change it/ add a new one, but wasn't sure if there was a reason it is where it is currently.

Reply:
Yeah I don't know where that path came from, but I think it makes more sense for the `data` directory to be at the top level.

Reply:
Just to be clear, the way I'm picturing this is that someone who wants to use this repo will download the netflix data, and extract it to `MediaBridge/data` and then we basically treat that directory as "read only". We don't write any files there, and we don't modify anything there. We don't touch it. That way, once the data is "installed" no one has to think about it ever again.

Reply:
Path should be `../../data`

Reply:
Also `base_dir` should be called `data_dir` or something.